### PR TITLE
Update DG, UG and ReadMe

### PR DESCRIPTION
**Pull request by DonovanJJ**
DG update:
- removing of class
- updating class note
- adding lesson to class schedule
- removing lesson from class schedule

UG update:
- removing of class
- Just a header for all remaining features for v1.2

Readme update:
- Updating the Readme paragraphs to suit our project

### CODE REVIEW

**Review by sudarshan2401 — APPROVED** [review]
LGTM! Thanks for updating the README

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -295,30 +295,146 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to view the list of classes
+2.  EduTrack shows a list of classes
+3.  User requests to delete a specific class in the list
+4.  EduTrack request for confirmation and waits for y/n response from user
```
Comment (seanpzk) [inline docs/DeveloperGuide.md]: I like how clear and concise your MSS is. I think there is a small grammar mistake for "request", should be "requests" instead 😄

**Review by seanpzk — APPROVED** [review]
LGTM 👍

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (DonovanJJ) [reply docs/DeveloperGuide.md]: Thanks for spotting this mistake 👍 Its been fixed

**Pull request merged**